wire-library/wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/internal/parser/OptionReader.kt

```diff
@@ -69,7 +69,7 @@ class OptionReader(internal val reader: SyntaxReader) {
     val kindAndValue = readKindAndValue()
     var kind = kindAndValue.kind
     var value = kindAndValue.value
-    if (subName != null) {
+    subNames.reversed().forEach { subName ->
```

Comment:
we should talk about forEach() vs. for()

Reply:
Yes ! I know you're for the latter...